test/position_helper_test.rb

```diff
@@ -0,0 +1,68 @@
+# frozen_string_literal: true
+require "test_helper"
+
+module ThemeCheck
+  module LanguageServer
+    class PositionHelperTest < Minitest::Test
+      def setup
+        @helper = Object.new
+        @helper.extend(PositionHelper)
```

Comment:
You could just `include PositionHelper` in this class, and use the methods without any receiver.